client/src/components/Dashboard.js

```diff
@@ -107,9 +108,29 @@ class Dashboard extends Component {
                   translation_id
                 } = document;
                 const dueDate = moment(document.due_date).format("L");
+                const currentDate = new Date();
+                const dateOnly = moment(currentDate).format("L");
+                const threeDaysLeft = moment(currentDate)
+                  .add(3, "days")
+                  .format("L");
+                const twoDaysLeft = moment(currentDate)
+                  .add(2, "days")
+                  .format("L");
+                const oneDayLeft = moment(currentDate)
+                  .add(1, "days")
+                  .format("L");
```

Comment:
All of these variable don't need to be defined here as they are only used inside NameColumn component. You can just pass "currentDate" as a props to NameColumn component and inside the component define all these variables.

Reply:
ok